src/main/java/sqlline/SqlLinePropertiesEnum.java

```diff
@@ -114,6 +114,24 @@ public static SqlLineProperty valueOf(
     }
     return null;
   }
+
+  /**
+   * Class to handle exception for {@link Application#getVersion()}.
+   */
+  private static class Constants {
+    static final String VERSION_VALUE;
+
+    static {
+      String ver;
+      try {
+        ver = new Application().getVersion();
+      } catch (Exception e) {
+        e.printStackTrace();
```

Comment:
We do not have sqlline instance at this stage, so that is the issue. Moreover `printstacktrace` is used inside of `Sqlline#handleException` sqlline instance is marked as `initComplete == false`.

ok will moved to the solution at dbb87d1